### PR TITLE
[android] Fixed Place Page opening behavior

### PR DESCRIPTION
**Pull request by goblinr**
Fixed Place Page opening behavior(https://jira.mail.ru/browse/MAPSME-2843).

@alexzatsepin 
@mpimenov

### CODE REVIEW

File: android/src/com/mapswithme/maps/widget/placepage/BottomPlacePageAnimationController.java [hunk 1]
```diff
@@ -329,7 +326,16 @@ public void run()
             hidePlacePage();
             break;
           case PREVIEW:
-            showPreview(currentState);
+            mPreview.addOnLayoutChangeListener(new View.OnLayoutChangeListener()
+            {
+              @Override
+              public void onLayoutChange(View v, int left, int top, int right, int bottom, int oldLeft, int oldTop, int oldRight, int oldBottom)
```
Comment (goblinr) [inline android/src/com/mapswithme/maps/widget/placepage/BottomPlacePageAnimationController.java]: Done

File: android/src/com/mapswithme/maps/widget/placepage/BottomPlacePageAnimationController.java [hunk 2]
```diff
@@ -169,15 +169,12 @@ public boolean onScroll(MotionEvent e1, MotionEvent e2, float distanceX, float d
           mIsDragging = true;
           if (!translateBy(-distanceY))
           {
-            if (mDetailsScroll.getTranslationY() == 0)
+            if ((isDetailContentScrollable() && mDetailsScroll.getTranslationY() == 0)
+                || (!isDetailContentScrollable() && mDetailsScroll.getTranslationY() <= mDetailsScroll.getHeight() - mDetailsContent.getHeight()))
```
Comment (goblinr) [inline android/src/com/mapswithme/maps/widget/placepage/BottomPlacePageAnimationController.java]: Done

**Comment by alexzatsepin** [comment]
LGTM

**Comment by alexzatsepin** [comment]
@mpimenov PTAL please

**Comment by mpimenov** [comment]
LGTM

**Pull request merged**